package.json

```diff
@@ -12,7 +12,7 @@
     "lint:fix": "npm run lint -- --fix",
     "e2e": "npm exec cypress install && npm exec grafana-e2e run",
     "e2e:update": "npm exec cypress install && npm exec grafana-e2e run --update-screenshots",
-    "server": "docker compose up --build",
+    "server": "docker-compose up --build",
```

Comment:
I see what you meant about the `docker-compose`, but this change was made by the update process

Reply:
I understand that it was made by the update process, but I'll have you change it back as `docker compose` is what I use in my workflow and I don't think anyone should try to use `docker-compose` in their workflows.